packages/features/webhooks/lib/sendPayload.ts

```diff
@@ -44,9 +44,39 @@ export type TranscriptionGeneratedPayload = {
   };
 };
 
+export type OOOEntryPayload = {
+  oooEntry?: {
+    id?: number;
+    start?: string;
+    end?: string;
+    createdAt?: string;
+    updatedAt?: string;
+    notes?: string | null;
+    reason?: {
+      emoji?: string;
+      reason?: string;
+    };
+    reasonId?: number;
+    user?: {
+      id?: number;
+      name?: string | null;
+      username?: string | null;
+      timeZone?: string;
+    };
+    toUser?: {
+      id?: number;
+      name?: string | null;
+      username?: string | null;
+      timeZone?: string;
+    } | null;
+    uuid?: string;
+  };
+};
+
 export type WebhookDataType = CalendarEvent &
   TranscriptionGeneratedPayload &
   // BookingNoShowUpdatedPayload & // This breaks all other webhooks
+  OOOEntryPayload &
```

Comment:
For OOO entries we don't need all the calendar event data. Can we somehow improve that type by using or ` | ` for  `OOOEntryPayload` instead?